protocol/tezos/harbinger-wrapper/src/__tests__/e2e/get-asset-data.spec.ts

```diff
@@ -81,40 +70,9 @@ describe("Query", () => {
         expect(response.data?.getAssetData.startPeriod).toBeDefined()
     })
 
-    it.skip("should get asset data for `XTZ-USD` on granadanet", async () => {
-      const response =  await client.query<{ getAssetData: QuerySchema.AssetCandle}>({
-        uri: ensUri,
-        query: `
-        query {
-          getAssetData(
-            assetCode: $assetCode,
-            network: granadanet,
-            providerAddress: $providerAddress
-            )
-          }
-          `,
-          variables: {
-            assetCode: "XTZ-USD",
-            providerAddress: "KT1ENR6CK7cBWCtZt1G3PovwTw3FgSW472mS"
-          }
-        })
-      
-      expect(response.errors).toBeUndefined()
-      expect(response.data).toBeDefined()
-      expect(response.data?.getAssetData).toBeDefined()
-      expect(response.data?.getAssetData.low).toBeDefined()
-      expect(response.data?.getAssetData.open).toBeDefined()
-      expect(response.data?.getAssetData.high).toBeDefined()
-      expect(response.data?.getAssetData.asset).toBeDefined()
-      expect(response.data?.getAssetData.close).toBeDefined()
-      expect(response.data?.getAssetData.volume).toBeDefined()
-      expect(response.data?.getAssetData.endPeriod).toBeDefined()
-      expect(response.data?.getAssetData.startPeriod).toBeDefined()
-    })
-
     it("should get asset data for `XTZ-USD` on custom network", async () => {
-      const response =  await client.query<{ getAssetData: QuerySchema.AssetCandle}>({
-        uri: ensUri,
+      const response =  await client.query<{ getAssetData: AssetCandle}>({
```

Comment:
don't use `client.query` anymore. I'd suggest to use app codegen whenever possible and if not then invoke.